src/erlazure_insights.erl

```diff
@@ -95,10 +106,10 @@ test_url_construction() ->
     
     ToDateTime = {{2016, 2, 3}, {14, 23, 57}},
 
-    ToTimestamp = get_time_offset_as_timestamp(ToDateTime, {seconds, 0}),
-    FromTimestamp = get_time_offset_as_timestamp(ToDateTime, {hours, 24}),
+    ToTimestamp = "To",
+    FromTimestamp = "From",
     Filter = create_time_bounded_filter(FromTimestamp, ToTimestamp),
-    Select = "I%20am%20selecting%20something",
+    Select = "I am selecting something",
```

Comment:
let's use real example there - `Select = "eventName, eventSource"`, as one of the fields that could be selecred